src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -1,299 +1,315 @@
+
+        /// <summary>
+        /// Gets or sets the package file(s) used for parsing and extracting relevant installer metadata.
+        /// </summary>
+        private readonly List<string> packageFiles = new();
+
+        /// <summary>
+        /// Gets the usage examples for the New command.
+        /// </summary>
+        [Usage(ApplicationAlias = ProgramApplicationAlias)]
+        public static IEnumerable<Example> Examples
+        {
+            get
+            {
+                yield return new Example(Resources.Example_NewCommand_StartFromScratch, new NewCommand { });
+                yield return new Example(Resources.Example_NewCommand_DownloadInstaller, new NewCommand { InstallerUrls = new string[] { "<InstallerUrl1>", "<InstallerUrl2>, .." } });
+                yield return new Example(Resources.Example_NewCommand_SaveLocallyOrSubmit, new NewCommand
+                {
+                    InstallerUrls = new string[] { "<InstallerUrl1>", "<InstallerUrl2>, .." },
+                    OutputDir = "<OutputDirectory>",
+                    GitHubToken = "<GitHubPersonalAccessToken>",
+                });
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the installer URL(s) used for downloading and parsing the installer file(s).
+        /// </summary>
+        [Value(0, MetaName = "urls", Required = false, HelpText = "InstallerUrl_HelpText", ResourceType = typeof(Resources))]
+        public IEnumerable<string> InstallerUrls { get; set; }
+
+        /// <summary>
+        /// Gets or sets the outputPath where the generated manifest file should be saved to.
+        /// </summary>
+        [Option('o', "out", Required = false, HelpText = "OutputDirectory_HelpText", ResourceType = typeof(Resources))]
+        public string OutputDir { get; set; }
+
+        /// <summary>
+        /// Executes the new command flow.
+        /// </summary>
+        /// <returns>Boolean representing success or fail of the command.</returns>
+        public override async Task<bool> Execute()
+        {
+            CommandExecutedEvent commandEvent = new CommandExecutedEvent
+            {
+                Command = nameof(NewCommand),
+                InstallerUrls = this.InstallerUrls,
+                HasGitHubToken = !string.IsNullOrEmpty(this.GitHubToken),
+            };
+
+            try
+            {
+                Prompt.Symbols.Done = new Symbol(string.Empty, string.Empty);
+                Prompt.Symbols.Prompt = new Symbol(string.Empty, string.Empty);
+
+                Manifests manifests = new Manifests();
+
+                if (!this.InstallerUrls.Any())
+                {
+                    this.InstallerUrls = PromptProperty(
+                        new Installer(),
+                        this.InstallerUrls,
+                        nameof(Installer.InstallerUrl));
                 }

```

Comment:
I believe this fails now because InstallerUrls is no longer just a string and is now a list of strings so there would be a casting error.

Reply:
Nice catch! This was not fun to fix :) Unfortunately, sharprompt doesn't move the cursor down low enough after the user enters a big block of text, so the next output show up on top of the input text. I got around that by just console.clear'ing after this